frontend/projects/valtimo/admin-settings/src/lib/components/admin-settings-menu-configuration/admin-settings-menu-configuration.component.scss

```
@@ -129,10 +129,15 @@
 
-  &__row-route {
+  &__row-subtitle {
     color: var(--cds-text-secondary);
-    font-size: var(--cds-code-01-font-size, var(--cds-body-compact-01-font-size));
-    font-family: var(--cds-code-01-font-family, monospace);
+    font-size: var(--cds-label-01-font-size);
+    letter-spacing: var(--cds-label-01-letter-spacing);
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    // See __row-title: keep the line box tall enough that descenders are not clipped by overflow.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)